articles/api-management/api-management-cross-domain-policies.md

```diff
@@ -54,7 +56,7 @@ Use the `cross-domain` policy to make the API accessible from Adobe Flash and Mi
 This policy can be used in the following policy [sections](./api-management-howto-policies.md#sections) and [scopes](./api-management-howto-policies.md#scopes).
 
 - **Policy sections:** inbound
-- **Policy scopes:** all scopes
+- **Policy scopes:** tenant
```

Comment:
I think this should be reverted to "all scopes". "Scope" here refers to the scope in API Management: product, API, operation, etc.

Reply:
I'm a dev on the APIM team and internally we use "tenant" instead of "global". 
I've just checked the public docs and you're right, it should be "global" in public docs. I've updated the PR.

![image](https://user-images.githubusercontent.com/8254243/157634840-82b88997-2439-4820-8cb3-70cc0a99d24f.png)
This is the APIM code that shows where the policy is valid and we don't allow the cross-domain policy to be used anywhere except for in the global (tenant) scope. The docs saying this policy can be used anywhere are wrong.

The example of the policy in the public docs was also invalid, so I've fixed that in this PR as well.